modules/core/src/main/java/org/apache/synapse/mediators/builtin/CallMediator.java

```diff
@@ -68,55 +75,90 @@ public boolean mediate(MessageContext synInCtx) {
 
         SynapseLog synLog = getLog(synInCtx);
 
-        synLog.traceOrDebug("Start : Call mediator");
-        if (synLog.isTraceTraceEnabled()) {
-            synLog.traceTrace("Message : " + synInCtx.getEnvelope());
-        }
-
-        boolean outOnlyMessage = "true".equals(synInCtx.getProperty(
-                SynapseConstants.OUT_ONLY));
+        if (!blocking) {
+            synLog.traceOrDebug("Start : Call mediator - Non Blocking Call");
```

Comment:
The code is conflict that is why,the above lines are displaying in wrong place.
the following code is added after the pull request. I deleted the following lines and merged the pull request with develop branch,after that I was able to merge the code.

// clear the message context properties related to endpoint in last service invocation
            Set keySet = synInCtx.getPropertyKeySet();
            if (keySet != null) {
                keySet.remove(SynapseConstants.RECEIVING_SEQUENCE);
                keySet.remove(EndpointDefinition.DYNAMIC_URL_VALUE);
            }